services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/utils/ThreadPoolExecutorUtils.java

```diff
@@ -0,0 +1,14 @@
+package com.futurewei.alcor.subnet.utils;
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+public class ThreadPoolExecutorUtils {
+    public static final ThreadPoolExecutor SELECT_POOL_EXECUTOR = new ThreadPoolExecutor(10, 20, 5000,
```

Comment:
what do those value (10,20,5000) stand for? 

A recommended way is to use "field name: field value". 

Also, if possibly, make those value configurable in our application.json file; if not possible, we should make those field as constants and maintain the constants in a separate file.

Reply:
corePoolSize:10. maximumPoolSize:20, KeepAliveTime:5000.
I agree with you, I will make those field as constants.